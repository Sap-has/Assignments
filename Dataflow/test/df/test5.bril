@main {
  x: int = const 10;
  cond: bool = const true;
  br cond .left .right;
.left:
  y: int = add x x;
  jmp .merge;
.right:
  y: int = mul x x;
  jmp .merge;
.merge:
  z: int = add y x;
  print z;
}