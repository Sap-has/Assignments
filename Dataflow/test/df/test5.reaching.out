b1:
  in:  âˆ…
  out: cond.b1.1, x.b1.0
left:
  in:  cond.b1.1, x.b1.0
  out: cond.b1.1, x.b1.0, y.left.0
right:
  in:  cond.b1.1, x.b1.0
  out: cond.b1.1, x.b1.0, y.right.0
merge:
  in:  cond.b1.1, x.b1.0, y.left.0, y.right.0
  out: cond.b1.1, x.b1.0, y.left.0, y.right.0, z.merge.0
